article/app/views/fragments/liveBlogNavigation.scala.html

```diff
@@ -15,7 +15,7 @@
             }
             class="liveblog-navigation__link liveblog-navigation__link--secondary liveblog-navigation__link--secondary--newer @if(pagination.newest.isEmpty) { liveblog-navigation__link--disabled }"
             data-link-name="newest page">
-                @fragments.inlineSvg("arrow-left-double", "icon", List("liveblog-navigation__link--secondary--newer__icon"))
+                @fragments.inlineSvg("arrow-left-double", "icon", List("liveblog-navigation__link--secondary--newer__icon", if(pagination.newest.isEmpty) "liveblog-navigation__link--disabled" else "liveblog-navigation__link--enabled"))
```

Comment:
no idea if I'm on the right track here, but I think the `a` is the `liveblog-navigation__link` so is it right from a BEM point of view to use the class on the svg too?  if the way it works in the standard won't work in amp, maybe we need to change both to work in the amp way?